share/getters/shrex_test.go

```diff
@@ -57,11 +58,12 @@ func TestShrexGetter(t *testing.T) {
 	require.NoError(t, getter.Start(ctx))
 
 	t.Run("ND_Available", func(t *testing.T) {
-		ctx, cancel := context.WithTimeout(ctx, time.Second)
+		ctx, cancel := context.WithTimeout(ctx, time.Second*10)
 		t.Cleanup(cancel)
 
 		// generate test data
-		randEDS, dah, namespace := generateTestEDS(t)
+		namespace := sharetest.RandV0Namespace()
+		randEDS, dah := edstest.RandEDSWithNamespace(t, namespace, 64)
```

Comment:
Can you make some comments about what this is testing here? Or make this an independent test within the table tests here that is named after what case it's testing.

Reply:
Update test name to highlight choice of big eds